code/modules/mob/living/basic/space_fauna/spider/spider_abilities/web.dm

```diff
@@ -62,7 +62,7 @@
 /datum/action/cooldown/lay_web/proc/plant_web(turf/target_turf, obj/structure/spider/stickyweb/existing_web)
 	new /obj/structure/spider/stickyweb(target_turf)
 
-/// Variant for genetics, created webs only allow the creator passage
+/// Variant for genetics, created webs only allow the creator snare
```

Comment:
This makes me think you did a blind replace all.

Reply:
I did indeed